### PR TITLE
Katacoda Broken Link Updated

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #34428

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-34496--kubernetes-io-main-staging.netlify.app/docs/tasks/run-application/run-replicated-stateful-application/#:~:text=these%20Kubernetes%20playgrounds%3A-,Katacoda,-Play%20with%20Kubernetes

File: content/en/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ cluster, you can create one by using
 [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
 or you can use one of these Kubernetes playgrounds:
 
-* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
+* [Katacoda](https://www.katacoda.com/tfogo/scenarios/k8s)
```
Comment (sftim) [inline content/en/includes/task-tutorial-prereqs.md:7]: I'm not sure that this is the right fix.

Do we know that https://www.katacoda.com/tfogo/scenarios/k8s will remain in place, and for how long? Katacoda is shutting down.
How much trust do we have in that user's private scenarios?

**Comment by NitishKumar06** [comment]
@sftim I think we could refer to Killercoda then. Let me raise a PR again and change the title as well.

**Comment by reylejano** [comment]
I'm ok with changing the link to Killercoda
/lgtm

**Comment by reylejano** [comment]
After some thought, I'm reconsidering my review.
The Katacoda scenarios for kubernetes.io are available to the public. 
This PR is to update the Katacoda link which should be 
https://www.katacoda.com/tfogo/scenarios/k8s

This PR can also add killercoda. If that [Katacoda scenario](https://www.katacoda.com/tfogo/scenarios/k8s) is appropriate, let's replace the current link with that
 
/lgtm cancel

**Comment by windsonsea** [comment]
Excuse me. Just mark a note here if this is merged.

**Comment by Sea-n** [comment]
The PR #34429 already fixed this issue.
/close

**Pull request closed without merging**